README.md

```diff
@@ -17,15 +17,36 @@ There are two `application.properties` files to edit:
 For testing the application locally, follow the next steps:
 1. Install `Java = 8` and `Maven`. 
 2. Install the Maven dependencies by running `mvn -Djdk.tls.client.protocols=TLSv1.2 clean install`. 
-3. (Optional) Download the latest ONE Record Turtle ontologies from Github + the latest Web Access Control from W3C by running `mvn -Djdk.tls.client.protocols=TLSv1.2 exec:java`.
+3. (Optional) If the error below occurs ([Stackoverflow explanation](https://stackoverflow.com/questions/67833372/getting-blocked-mirror-for-repositories-maven-error-even-after-adding-mirrors)) it is because Maven recently only allows to get dependencies via `https`.
+```
+[ERROR] Failed to execute goal on project one-record-server-java: 
+Could not resolve dependencies for project org.iata:one-record-server-java:jar:1.0-SNAPSHOT: 
+Failed to collect dependencies at cz.cvut.kbss.jopa:ontodriver-owlapi:jar:0.15.0 -> cz.cvut.kbss:owl2query-engine:jar:0.6.0: 
+Failed to read artifact descriptor for cz.cvut.kbss:owl2query-engine:jar:0.6.0: 
+Could not transfer artifact cz.cvut.kbss:owl2query-engine:pom:0.6.0 from/to maven-default-http-blocker (http://0.0.0.0/): transfer failed for http://0.0.0.0/cz/cvut/kbss/owl2query-engine/0.6.0/owl2query-engine-0.6.0.pom: 
+Connect to 0.0.0.0:80 [/0.0.0.0] failed: Connection refused -> [Help 1]
+```
+To solve the error just comment out the lines below in `[MAVEN_DIR]/conf/settings.xml`
+```
+<!--
+<mirror>
+  <id>maven-default-http-blocker</id>
+  <mirrorOf>external:http:*</mirrorOf>
+  <name>Pseudo repository to mirror external repositories initially using HTTP.</name>
+  <url>http://0.0.0.0/</url>
+  <blocked>false</blocked>
+</mirror>
+-->
+```
```

Comment:
Hello @Meschkov,

the `Connect to 0.0.0.0:80 [/0.0.0.0] failed` got already fixed with https://github.com/IATA-Cargo/one-record-server-java/pull/25 so maybe that part should be removed from the README.